_aggregations/metric/cardinality.md

```diff
@@ -9,9 +9,25 @@ redirect_from:
 
 # Cardinality aggregations
 
-The `cardinality` metric is a single-value metric aggregation that counts the number of unique or distinct values of a field.
+The `cardinality` metric is a single-value metric that counts the number of unique or distinct values of a field.
 
-The following example finds the number of unique products in an eCommerce store:
+
+Cardinality count is approximate. See [Controlling precision](#controlling-precision).
+
+## Parameters
+
+The `cardinality` aggregation takes the following parameters: 
+
+| Parameter             | Required/Optional | Data type       | Description |
+| :--                   | :--               |  :--            | :--         |
+| `field`               | Required          | String          | The field for which the cardinality is estimated. |
+| `precision_threshold` | Optional          | Numeric         | Count below which good accuracy is expected.      |
+| `execution_hint`      | Optional          | String          | How to run the aggregation. Allowed values are `ordinals` and `direct`. |
```

Comment:
```suggestion
| `execution_hint`      | Optional          | String          | How to run the aggregation. Valid values are `ordinals` and `direct`. |
```

Reply:
Should we clarify what `ordinals` and `direct` mean or is it documented somewhere already?